lib/trino-filesystem/src/main/java/io/trino/filesystem/tracing/TracingOutputFile.java

```diff
@@ -88,13 +88,13 @@ public OutputStream createOrOverwrite(AggregatedMemoryContext memoryContext)
     }
 
     @Override
-    public OutputStream createExclusive(AggregatedMemoryContext memoryContext)
+    public void createExclusive(byte[] content, AggregatedMemoryContext memoryContext)
             throws IOException
     {
-        Span span = tracer.spanBuilder("OutputFile.createExclusive")
+        Span span = tracer.spanBuilder("OutputFile.createAtomic")
```

Comment:
why rename span name and not the method?

Reply:
fixed